selfdrive/car/hyundai/values.py

```diff
@@ -51,7 +51,7 @@ class CAR:
   SONATA_LF = "HYUNDAI SONATA 2019"
   PALISADE = "HYUNDAI PALISADE 2020"
   VELOSTER = "HYUNDAI VELOSTER 2019"
-  SONATA_HYBRID = "HYUNDAI SONATA HYBRID 2021"
+  SONATA_HYBRID = "HYUNDAI SONATA HYBRID 2020-22"
```

Comment:
Make sure not to touch the CAR entries for car ports, these are used for fingerprinting and offline to process the logs

Reply:
Yup, you only need to update the CarInfo